code/game/objects/machinery/overwatch.dm

```diff
@@ -658,6 +663,18 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 	visible_message(span_boldnotice("[transfer_marine] has been transfered from squad '[old_squad]' to squad '[new_squad]'. Logging to enlistment file."))
 	to_chat(transfer_marine, "[icon2html(src, transfer_marine)] <font size='3' color='blue'><B>\[Overwatch\]:</b> You've been transfered to [new_squad]!</font>")
 
+///Big light :)
+/obj/machinery/computer/camera_advanced/overwatch/proc/attempt_spotlight(datum/source, atom/A, params)
+	SIGNAL_HANDLER
+	if(TIMER_COOLDOWN_CHECK(src, COOLDOWN_ORBITAL_SPOTLIGHT))
+		to_chat(source, span_notice("The Orbital spotlight is still recharging."))
+		return
+	var/turf/target = get_turf(A)
+	target.set_light(11, l_color = COLOR_TESLA_BLUE)
+	playsound(target,'sound/mecha/heavylightswitch.ogg', 25, 1, 20)
+	to_chat(target, span_warning("You see a twinkle in the sky before your surroundings are hit with a beam of light!"))
+	to_chat(source, span_notice("Orbital spotlight activated. Duration : [SPOTLIGHT_DURATION]"))
+	addtimer(CALLBACK(target, /atom.proc/set_light, 0, 0), SPOTLIGHT_DURATION)
```

Comment:
what happens if you illuminate lava?

Reply:
Imo it would be better to have like a holder for the light, rather than changing the lights of the turf

Reply:
I used an effect to skirt around this issue